samples/Sentry.Samples.AspNetCore.Blazor.Wasm/Program.cs

```diff
@@ -1,20 +1,19 @@
-using System.Diagnostics;
 using Microsoft.AspNetCore.Components.WebAssembly.Hosting;
 using Sentry.Samples.AspNetCore.Blazor.Wasm;
 
-// Capture blazor bootstrapping errors
-using var sdk = SentrySdk.Init(o =>
-{
-    o.Dsn = "https://eb18e953812b41c3aeb042e666fd3b5c@o447951.ingest.sentry.io/5428537";
-    o.Debug = true;
-    //IsGlobalModeEnabled will be true for Blazor WASM
-    Debug.Assert(o.IsGlobalModeEnabled);
-});
 try
 {
     var builder = WebAssemblyHostBuilder.CreateDefault(args);
+    builder.UseSentry(options =>
+    {
+        options.Dsn = "https://eb18e953812b41c3aeb042e666fd3b5c@o447951.ingest.sentry.io/5428537";
+        options.Debug = true;
+        options.IsGlobalModeEnabled = true;
```

Comment:
Should this be set automagically? Ideally this isnt' a decision the user needs to make

Reply:
Good point!